script_files/localStorageHandler.js

```diff
@@ -0,0 +1,108 @@
+export function handleLocalStorage(localStorage) {
```

Comment:
```suggestion
export function parseLocalStorage(localStorage) {
```

This seems like a more fitting name :)